elyra/pipeline/processor_kfp.py

```diff
@@ -67,8 +97,22 @@ def process(self, pipeline):
             self.log.info("Kubeflow Pipeline successfully compiled.")
             self.log.debug("Kubeflow Pipeline was created in %s", pipeline_path)
 
-            # Upload the compiled pipeline and create an experiment and run
-            client = kfp.Client(host=api_endpoint)
+            # Upload the compiled pipeline, create an experiment and run
+
+            if (api_endpoint_username is not None) and (api_endpoint_password is not None):
```

Comment:
you can do similar to JS/TS here if(api_endpoint_username) which will cover for None and empty strings